media-libs/grok/grok-9.7.6.ebuild

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
EAPI 8?

Reply:
Probably yes, but can't test this at the moment in my environment